blueocean-jwt/src/main/java/io/jenkins/blueocean/auth/jwt/JwtAuthenticationService.java

```diff
@@ -14,7 +14,7 @@
  *
  * @author Vivek Pandey
  */
-public abstract class JwtAuthenticationService implements RootAction, ExtensionPoint{
+public abstract class JwtAuthenticationService implements UnprotectedRootAction, ExtensionPoint{
```

Comment:
Yeah, there is test that calls token generation and wks service. See [here](https://github.com/jenkinsci/blueocean-plugin/blob/bs-poc-pr/blueocean-jwt/src/test/java/io/jenkins/blueocean/auth/jwt/impl/JwtAuthenticationServiceImplTest.java#L29).